interface/patient_file/reminder/clinical_reminders.php

```diff
@@ -132,7 +134,7 @@
       </table>
       <br />
       <br />
-      <table>
+      <table class="table table-bordered table-active table-hover">
```

Comment:
Surround table with a div element that has a class of `table-responsive` ([reference](https://getbootstrap.com/docs/4.4/content/tables/#responsive-tables))

Reply:
on it, thank you for the review :)